proxywasm/hostcall_test.go

```diff
@@ -17,22 +17,26 @@ package proxywasm
 import (
 	"testing"
 
+	"github.com/stretchr/testify/assert"
```

Comment:
it is intentional that I haven't used`assert` pkg anywhere, and stick to require, so could you use `require`, not `assert`

Reply:
Sure, can do. May I ask why?

`require` stops the execution at the first failure which would hide further failures in the test until fixed and run again.

Reply:
This has been addressed [here](https://github.com/tetratelabs/proxy-wasm-go-sdk/pull/302/commits/9ec3010bd6ffc046082a7f323a81f7bdb23b3f0e).